docs/first-pr.md

```diff
@@ -632,7 +632,7 @@ Here's a checklist of action items to keep you sane:
 [production-ready code]: #production-ready-code
 [Reviewable]: https://reviewable.io
 [RocksDB]: http://rocksdb.org
-[STYLE.md]: /STYLE.md
+[style.md]: /docs/style.md
```

Comment:
All the links to `style.md` in this file should be done in the same way (relative vs absolute). Mind changing this to just `style.md` match the others?